templates/github/.github/workflows/publish.yml.j2

```diff
@@ -56,10 +56,10 @@ jobs:
 
       {{ run_script(name="Install", file="install.sh") | indent(6) }}
 
-      {{ run_script(name="Install Python client", file="install_python_client.sh", withenv=False) | indent(6) }}
+      {{ run_script(name="Build Python bindings packages", file="build_python_client.sh", withenv=False) | indent(6) }}
 
       {%- if deploy_client_to_rubygems %}
-      {{ run_script(name="Install Ruby client", file="install_ruby_client.sh", withenv=False) | indent(6) }}
+      {{ run_script(name="Build Ruby bindings packages", file="build_ruby_client.sh", withenv=False) | indent(6) }}
```

Comment:
Same here. Actually the whole build-bindings-docs should have been done by build already. Can you completely remove that job from the workflow?

Reply:
Sure, let me try that.